src/components/avatar/Avatar.js

```diff
@@ -20,6 +22,10 @@ type Props = {
     className?: string,
     /** Users id */
     id?: string | number,
+    /** force loading state if avatarUrl is not passed
+     *  (useful if avatar url is determined asynchronously to avoid flashing background color)
+     */
+    isPending?: boolean,
```

Comment:
Can this be achieved without isPending? By using key prop with url value? When URL is undefined/null, then always show the flashing thingy, and when url is there, then show the avatar.

Reply:
The pending state is for when the URL has not been determined; the wrapper avatar does an async call to get an authenticated download URL.

Reply:
One other idea is to merge the implementation of the element avatar (with `getAvatarUrl`) and the component (with `isPending`), since the `isPending` prop exists to support the state of the elements avatars that resolve async.